README.md

```diff
@@ -1,39 +1,60 @@
 # "Hello world"? We prefer "Hello Bao!"
 
-Welcome to the Bao Hypervisor! Get ready for an interactive journey as we explore the world of Bao together. Whether you're a seasoned Bao user or a newcomer, this tour is designed to give you a practical and enthusiastic introduction to our powerful hypervisor.
+Welcome to the Bao Hypervisor! Get ready for an interactive journey as we
+explore the world of Bao together. Whether you're a seasoned Bao user or a
+newcomer, this tour is designed to give you a practical and enthusiastic
+introduction to our powerful hypervisor.
 
-If you're already familiar with Bao or want to dive into specific setups provided by our team, feel free to skip ahead to the Bao demos section.
+If you're already familiar with Bao or want to dive into specific setups
+provided by our team, feel free to skip ahead to the Bao demos section.
```

Comment:
If you mean a section inside of the README, please link it here.

Reply:
Done!